react/Critical/Critical.less

```diff
@@ -1,5 +1,5 @@
-@import (reference) "~seek-style-guide/theme";
+@import (reference) '~seek-style-guide/theme';
```

Comment:
Is this another Prettier update? Do we need to bite the bullet and add Prettier to this repo? 😄

Reply:
Oh yeah it is - I missed that change when I pushed it up. I have it turned on in my editor. I would love to have prettier on in this repo

Reply:
Please do bite that bullet.